src/goto-programs/goto_convert_functions.cpp

```diff
@@ -155,14 +153,6 @@ void goto_convert_functionst::convert_function(
      symbol.value.id()=="compiled") /* goto_inline may have removed the body */
     return;
 
-  if(symbol.value.id()!=ID_code)
-  {
-    error().source_location=symbol.value.find_source_location();
-    error() << "got invalid code for function `" << identifier << "'"
-            << eom;
-    throw 0;
-  }
-
```

Comment:
Actually, I think I'm mistaken about removing this error case... I'll revisit it.

Reply:
So I removed this throw case here because the same condition is immediately checked in the following `to_code(symbol.value)` statement. My only remaining doubt is whether this throw is actually checking user input or not... Thoughts @peterschrammel @kroening ?